iTerm2XCTests/SelectorSwizzler.h

```diff
@@ -0,0 +1,22 @@
+@import Foundation;
+
+/**
+ * A helper class for swizzling a selector for the scope of a block.
+ */
+@interface SelectorSwizzler : NSObject
```

Comment:
New classes should be named like `iTermSelectorSwizzler` (there are a some holdouts that I haven't renamed yet, but I'm moving toward this scheme.)